share/man/man4/acpi_wmi.4

```diff
@@ -22,47 +25,45 @@
 .\" OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 .\" SUCH DAMAGE.
 .\"
-.Dd September 5, 2019
+.Dd October 17, 2024
 .Dt ACPI_WMI 4
 .Os
 .Sh NAME
 .Nm acpi_wmi
-.Nd "ACPI to WMI mapping driver"
+.Nd ACPI to WMI mapping driver
 .Sh SYNOPSIS
 To compile this driver into the kernel,
 place the following line in your
 kernel configuration file:
-.Bd -ragged -offset indent
-.Cd "device acpi_wmi"
-.Ed
+.Pp
+.Dl Cd "device acpi_wmi"
 .Pp
 Alternatively, to load the driver as a
 module at boot time, place the following line in
 .Xr loader.conf 5 :
-.Bd -literal -offset indent
-acpi_wmi_load="YES"
-.Ed
+.Pp
+.Dl acpi_wmi_load="YES"
 .Sh DESCRIPTION
 The
 .Nm
 driver provides an interface for vendor specific WMI implementations
-(e.g. HP and Acer laptops).
-It creates /dev/wmistat%d, which can be read to get
-information about GUIDs found in the system.
+.Pq e.g. HP and Acer laptops .
+It creates
+.Pa /dev/wmistat%d ,
+which can be read to get information about GUIDs found in the system.
 .Sh FILES
 .Bl -tag -width /dev/wmistat%d -compact
 .It Pa /dev/wmistat%d
-WMI status device.
+WMI status devices.
 .El
 .Sh SYSCTLS
 The following sysctl node is currently implemented:
-.Bl -tag 
+.Bl -tag -width "dev.acpi_wmi.%d.bmof"
```

Comment:
I'd be tempted to leave this as `indent` (which is that man does when it's missing).

Sure, that "wastes" a line of text (by having a newline which may or may not be strictly necessary), but it's quite legible.

But I'd also be fine with making the width what you did, and letting it get rendered as it does.

(It's possible that I misunderstood your question in https://github.com/freebsd/freebsd-src/pull/1473#discussion_r1826239711 ?)